cmd/node/config/external.toml

```diff
@@ -44,13 +44,16 @@
 [HostDriverConfig]
     # This flag shall only be used for observer nodes
     Enabled = false
-    # This flag will start the WebSocket connector as server or client
-    IsServer = false
-    # The url of the WebSocket client/server
+    # This flag will start the WebSocket connector as server or client( can be "client" or "server")
+    Mode = "client"
+    # URL for the WebSocket client/server connection
+    # This value represents the IP address and port number that the WebSocket client or server will use to establish a connection.
     URL = "127.0.0.1:22111"
+    # After a message will be sent it will wait for an ack message if this flag is enabled
     WithAcknowledge = true
     # Currently, only "json" is supported. In the future, "gogo protobuf" could also be supported
     MarshallerType = "json"
     # The number of seconds when the client will try again to send the data
     RetryDurationInSec = 5
+    # Signals if in case of data payload processing error, we should send the ack signal or not
```

Comment:
```
Sets if, in case of data payload processing error, we should block or not the advancement to the next processing event. Set this to true if you wish the node to stop processing blocks if the client/server encounters errors while processing requests.
```

Reply:
done